### PR TITLE
Added configuration to generate c# sdk for CosmosDB service.

### PR DESCRIPTION
**Pull request by MehaKaushik**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by MehaKaushik** [comment]
@yungezz Can you please have a look at this PR. It is not a very big change. Shouldn't take a lot of time.

File: specification/cosmos-db/resource-manager/readme.md [hunk 1]
```diff
@@ -134,14 +134,30 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
+  - repo: azure-sdk-for-net
   - repo: azure-sdk-for-python
   - repo: azure-sdk-for-java
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-node
+  - repo: azure-sdk-for-ruby
```
Comment (yungezz) [inline specification/cosmos-db/resource-manager/readme.md]: pls not update this part, this is configuration for our internal CI tool, not for SDK generation.

**Comment by yungezz** [comment]
@isra-fel  could you pls look at this PR also since it's C# config? thanks.

**Comment by isra-fel** [comment]
C# part looks good to me.

**Comment by yungezz** [comment]
@MehaKaushik once you removed internal config change, and CI passed, the PR is  ok to merge.

**Comment by yungezz** [comment]
hi @MehaKaushik once change on internal config mentioned in prevoius comments revert, the pr is ok to merge

File: specification/cosmos-db/resource-manager/readme.md (hunk 1 above)
Comment (MehaKaushik) [reply specification/cosmos-db/resource-manager/readme.md]: removed it.

**Comment by MehaKaushik** [comment]
@yungezz made the required change. Apologies for delay, OOF previously.

**Review by yungezz — APPROVED** [review]

**Pull request merged**